Add index manually

sagangwee opened the pull request:
## Overview
<!--
READ ME FIRST:
Please answer *both* questions below and check off every point from the Essential Checklist!
If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in 1.
-->

1. This PR fixes or fixes part of #[fill_in_number_here].
2. This PR does the following: [Explain here what your PR does and why]

## Essential Checklist

- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ ] The linter/Karma presubmit checks have passed locally on your machine.
- [ ] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
  - This lets reviewers restart your CircleCI tests for you.
- [ ] The PR is made from a branch that's **not** called "develop".

## PR Pointers

- Make sure to follow the [instructions for making a code change](https://github.com/oppia/oppia/wiki/Contributing-code-to-Oppia#instructions-for-making-a-code-change).
- Oppiabot will notify you when you don't add a PR_CHANGELOG label. If you are unable to do so, please @-mention a code owner (who will be in the Reviewers list), or ask on [Gitter](https://gitter.im/oppia/oppia-chat).
- For what code owners will expect, see the [Code Owner's wiki page](https://github.com/oppia/oppia/wiki/Oppia%27s-code-owners-and-checks-to-be-carried-out-by-developers).
- Make sure your PR follows conventions in the [style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), otherwise this will lead to review delays.
- Never force push. If you do, your PR will be closed.

sagangwee committed:
Add index manually

sagangwee committed:
Add both index

seanlip commented:
Note: before merging, please try and generate these the usual way instead, so that these don't stand out as outliers. Thanks!

sagangwee committed:
Add moar indexes

seanlip reviewed (changes requested):
(Actually, putting a review hold on this one because it still needs a "proper" fix to autogenerate the indexes correctly.)

sagangwee commented:
Auto-regenerated by running jobs locally on a dev server in https://github.com/oppia/oppia/pull/11063. Original stack traces were of the following form:

```
Exception raised: no matching index found. recommended index is: (/base/data/home/apps/s~oppiaserver-backup-migration/test-sagang-cd.430390629934686904/core/controllers/base.py:477)

- kind: GeneralSuggestionModel properties: - name: language_code - name: last_updated

The suggested index for this query is:

- kind: GeneralSuggestionModel properties: - name: deleted - name: language_code - name: status - name: suggestion_type - name: last_updated
```

Seems only the index after `The suggested index for this query is:` was needed (per stack trace, as there were two total like the above). For posterity the second stack trace was:

```
Exception raised: no matching index found. recommended index is: (/core/controllers/base.py:477)

- kind: GeneralSuggestionModel properties: - name: suggestion_type - name: last_updated

The suggested index for this query is:

- kind: GeneralSuggestionModel properties: - name: deleted - name: status - name: suggestion_type - name: last_updated
``` 

Both errors were encountered after running the `CronMailReviewersContributorDashboardSuggestionsHandler` job on the backup server.
